.github/workflows/ci.yml

```diff
@@ -93,28 +143,6 @@ jobs:
           POSTGRES_PASSWORD: merlin
         run: make it-test-api-ci
 
-  publish-python-sdk:
-    if: ${{ startsWith(github.ref, 'refs/tags/') }}
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: 3.7
-      - name: Install dependencies
-        working-directory: ./python/sdk
-        run: |
-          python -m pip install --upgrade pip
-          pip install setuptools wheel twine
-      - name: Build and publish
-        env:
-          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-        working-directory: ./python/sdk
-        run: |
-          python setup.py sdist bdist_wheel
-          twine upload dist/*
-
   publish-merlin-docker:
```

Comment:
Why is there no `if` condition here while other publish jobs have? If this jobs is triggered and run from a fork, to which GHCR account the docker will be published? GitHub or accounts who forked the repo?

Reply:
Ah yes, good catch. The `if` condition should be here as well. To answer your question, the same `${{ secrets.GHCR_TOKEN }}` will be used to push the docker from fork repository based on the `if` condition.